add-on/src/lib/ipfs-companion.js

```diff
@@ -473,7 +459,7 @@ export default async function init () {
   // -------------------------------------------------------------------
 
   async function updateBrowserActionBadge () {
-    if (typeof browser.browserAction.setBadgeBackgroundColor === 'undefined') {
+    if (typeof browser.action.setBadgeBackgroundColor === 'undefined') {
       // Firefox for Android does not have this UI, so we just skip it
       return
     }
```

Comment:
It looks like this is supported in firefox for android now, https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/action/setBadgeBackgroundColor#browser_compatibility, so we can remove the content, but we may still want to double check for the existence of this method